CONTRIBUTING.rst

```diff
@@ -20,8 +20,11 @@ Making Your First Change
 ========================
 
 First, install xonsh from source and open a xonsh shell in your favorite
-terminal application. See installation instructions for details.
+terminal application. See installation instructions for details, but it
+is recommended to do an 'editable' install via `pip'
 
+        $ pip install -e .
```

Comment:
Previously new parser tables were not generated when doing an editable install. 

So for development on xonsh it is better to do a full install after changing stuff.

We can change this later.

Reply:
I've been using editable installs (`pip install --no-use-pep517 --user -e .`) since forever and it always re-generates the parser tables for me.

Reply:
Ahh. Yes. Of course. But you still need to rerun it to recreate the tables.